tests/validators/test_regexp.py

```diff
@@ -48,10 +48,29 @@ def test_regex_raises(re_pattern, re_flags, test_v, dummy_form, dummy_field):
         validator(dummy_form, dummy_field)
 
 
+def test_regexp_message_default(dummy_form, dummy_field, grab_error_message):
+    """
+    Regexp validator should return default message
+    """
+    validator = regexp("^a")
+    dummy_field.data = "f"
+    assert grab_error_message(validator, dummy_form, dummy_field) == "Invalid input."
+
+
 def test_regexp_message(dummy_form, dummy_field, grab_error_message):
     """
     Regexp validator should return given message
     """
     validator = regexp("^a", message="foo")
     dummy_field.data = "f"
     assert grab_error_message(validator, dummy_form, dummy_field) == "foo"
+
+
+def test_regexp_pattern_html(dummy_form, dummy_field):
```

Comment:
I am not sure of the point of this test. I suppose the assertion could go on `test_regex_passes`.